lib/pay/billable/stripe.rb

```diff
@@ -3,7 +3,9 @@ module Billable
     module Stripe
       def stripe_customer
         if processor_id?
-          customer = ::Stripe::Customer.retrieve(processor_id)
+         customer = ::Stripe::Customer.retrieve(processor_id)
+         customer.source = card_token 
+         customer.save
```

Comment:
Instead of setting the source, we need to call the `update_card` method, but only if the card_token is present.

https://github.com/jasoncharnes/pay/blob/master/lib/pay/billable.rb#L29

So more like:
`update_card(card_token) if card_token.present?`

This does mean the method is consistent in both cases. As long as you have the `card_token` set before this, it will either update the card for existing customers, or it will create a new customer with that card. I think this is correct for what we want here.